core/platform/datastore/gae_datastore_services.py

```diff
@@ -19,9 +19,34 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+from google.appengine.datastore import datastore_query
 from google.appengine.ext import ndb
 
 
+def make_cursor(urlsafe=None):
+    """Returns a database cursor that represents a relative position in a query.
+
+    The position denoted by a Cursor is relative to a result in a query even
+    if the result has been removed from the given query. Usually to position
```

Comment:
I don't understand the second sentence in this paragraph. Could you reword it to be clearer?

Reply:
How's that?